R/http.R

```diff
@@ -265,7 +267,14 @@ readHttpResponse <- function(request, conn) {
   storeCookies(request, setCookies)
 
   # read the response content
-  content <- rawToChar(readBin(conn, what = 'raw', n=contentLength))
+  if (is.null(contentLength)) {
+    # content length is unknown, so stream remaining text
+    content <- paste(readLines(con = conn), collapse = "\n")
```

Comment:
Is there any chance that some day in the future we might start getting binary content back with chunked encoding?

Reply:
That's a good question; I don't think so. This method is used to read API responses, which are uniformly text/html or application/json. We could further gate this on the content type if we had to but I don't think it's necessary.